modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -145,7 +147,11 @@ public class CatalogUtils {
         FUNCTIONAL_DEFAULT_FUNCTIONS.put("RAND_UUID", ColumnType.UUID);
     }
 
-    public static final List<String> SYSTEM_SCHEMAS = List.of(SYSTEM_SCHEMA_NAME);
+    public static final List<String> SYSTEM_SCHEMAS = List.of(
```

Comment:
I think, it worth to convert this collection to `Set`, wdyt?

Reply:
Done, thanks